pom.xml

```diff
@@ -1369,18 +1369,14 @@
         </reportSets>
       </plugin>
 
-      <!--
-       dsm-maven plugin does not support Java 1.8
-       Deactivated until https://github.com/sevntu-checkstyle/dsm-maven-plugin/issues/31
-       -->
-      <!--<plugin>-->
-        <!--<groupId>com.github.sevntu-checkstyle</groupId>-->
-        <!--<artifactId>dsm-maven-plugin</artifactId>-->
-        <!--<version>2.1.4</version>-->
-        <!--<configuration>-->
-          <!--<obfuscatePackageNames>true</obfuscatePackageNames>-->
-        <!--</configuration>-->
-      <!--</plugin>-->
+      <plugin>
+        <groupId>com.github.sevntu-checkstyle</groupId>
+        <artifactId>dsm-maven-plugin</artifactId>
+        <version>2.2.0</version>
```

Comment:
Is this executed as part of the `verify` phase?
If so, do we need to add something for the profile `no-validation`?

Reply:
it is just a report generation, no verification logic in this plugin for now.